extensions/ns-nl-search/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "ns-nl-search",
+  "title": "Nederlandse Spoorwegen Train Search",
+  "description": "Met de Raycast-extensie voor Nederlandse Spoorwegen kunt u eenvoudig treinschema's, routes en real-time updates voor het Nederlandse spoorwegsysteem vinden. Deze extensie vereenvoudigt het proces van het plannen van uw reis door snelle toegang te bieden tot essentiële informatie direct vanuit uw Raycast-interface. Of u nu pendelt of reist, blijf op de hoogte met de laatste treinupdates en zorg voor een soepele reis met slechts een paar klikken.\n\n\nWith the Raycast extension for Nederlandse Spoorwegen, you can easily find train schedules, routes, and real-time updates for the Dutch railway system. This extension simplifies the process of planning your journey by providing quick access to essential information directly from your Raycast interface. Whether you're commuting or traveling, stay informed with the latest train updates and ensure a smooth trip with just a few clicks.",
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension. If the locale might affect functionality (e.g. using the correct unit of measurement), please use the preferences API.